app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,38 @@
+<nav class="navbar navbar-expand-lg navbar-dark sticky-top" style="background-color: #036ffc;">
+  <a class="navbar-brand" href="#">Facebook Clone</a>
```

Comment:
This should link up to the root path, otherwise how can a user get back to that view? I was in `/users/index` and had no way of navigating back without pressing my browsers back button.

[`OPTIONAL`]